tests/molecules/LiH_dimer_pp/CMakeLists.txt

```diff
@@ -89,6 +89,55 @@ IF (NOT QMC_CUDA)
                     0 LIH_PP_XML_SCALARS # VMC
                     )
 
+
+#
+# Deterministic tests
+#
+
+# LiH molecular dimer pseudopotential tests
+  LIST(APPEND DET_LIH_PP_SCALARS "kinetic" "0.63652296 0.000001")
+  LIST(APPEND DET_LIH_PP_SCALARS "totenergy" "-0.75148455 0.000001")
+  LIST(APPEND DET_LIH_PP_SCALARS "eeenergy" "0.48090104 0.000001")
+  LIST(APPEND DET_LIH_PP_SCALARS "potential" "-1.38800751 0.000001")
+  LIST(APPEND DET_LIH_PP_SCALARS "samples" "1600000 0.0")
+
```

Comment:
Keep samples check in place here, but update the value here once "samples" in the input file is removed.